lib/db.js

```diff
@@ -434,6 +434,7 @@ module.exports = function (
             return bufferize({
               id: item.id,
               sessionToken: item.sessionTokenId,
+              lastConnectedAt: item.lastAccessTime,
```

Comment:
For some reason I *really* want to bikeshed the name `lastConnectedAt` here...but I'm not going to, it's pure aesthetics :-)